railties/lib/rails/test_help.rb

```diff
@@ -14,10 +14,12 @@
 if defined?(ActiveRecord::Base)
   ActiveRecord::Migration.maintain_test_schema!
 
-  class ActiveSupport::TestCase
-    include ActiveRecord::TestFixtures
-    self.fixture_path = "#{Rails.root}/test/fixtures/"
-    self.file_fixture_path = fixture_path + "files"
+  module ActiveSupport
+    class TestCase
```

Comment:
I took the liberty to expand this here so that the `TestCase` class won't be present at the top level of the API site's sidebar. Blaming this line points to https://github.com/rails/rails/commit/b17eb65d00242ae10ac9ed97ef22d88fdd710533 so since this is not a hot spot (the commit is 8 years old), this may not be make blames harder for this particular place.